CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Version 5.0.0
+
+- TagValues will be casted to theire types by default
+  - use `parseTagValue` option of parserOptions to change behaviour
+- Labels with whitespaces have to use `<![CDATA[ ]]>` when using whitespace
```

Comment:
This is very odd. Is there no way of having this more convenient? Does this only apply to leading or dangling whitespace? If so the note should be more specific